index.js

```diff
@@ -1,21 +1,24 @@
 'use strict';
 var stdin = process.stdin;
 
-module.exports = function () {
+module.exports = function (opt) {
 	var ret = '';
+	var tty = (opt && 'tty' in opt) ? opt.tty : module.exports.tty;
```

Comment:
Add `opts = opts || {}` above `var ret = ''` and change this to `var tty = 'tty' in opts ? opts.tty : module.exports.tty;`